src/platformIcon.tsx

```diff
@@ -47,7 +47,7 @@ export const PLATFORM_TO_ICON = {
   dotnetfx: "dotnetfx",
   dotnetframework: "dotnetframework",
   edge: "edge",
-  edgelegacy: "edge-legacy",
```

Comment:
Does anything depend on the previous keys? If so, maybe we should roll this out incrementally

1. add new key
2. roll it out
3. replace the usage whereever
4. remove old key
5. roll it out

Reply:
All these keys are from a newer batch of icons that aren't being used yet